src/python/pants/backend/python/dependency_inference/rules.py

```diff
@@ -225,7 +295,7 @@ async def infer_python_dependencies_via_imports(
                 "One or more unowned dependencies detected. Check logs for more details."
```

Comment:
(I can't seem to open an issue on line 282 :angry: )

If I'm being a pedant, the name `unowned_dependency_behavior` is now misleading, as files/resources are dependencies, but we just toss the inferred ones that don't have an owner.

A more correct name would be `unowned_import_behavior`. But also :man_shrugging: